spacemesh/v1/post.proto

```diff
@@ -0,0 +1,12 @@
+syntax = "proto3";
+package spacemesh.v1;
+option go_package = "github.com/spacemeshos/api/release/go/spacemesh/v1";
+
+import "spacemesh/v1/post_types.proto";
+
+// PoSTService is used by dedicated PoST nodes to interact with the spacemesh
+// node.
+service PoSTService {
```

Comment:
The [style guide](https://protobuf.dev/programming-guides/style/#services) suggests using PascalCase. Using `PoST` leads to weird generated code like module name: `spacemesh_v1::po_st_service_client`.

```suggestion
// PostService is used by dedicated PoST nodes to interact with the spacemesh
// node.
service PostService {
```